tools/jit/gen_jit_dispatch.py

```diff
@@ -322,6 +326,14 @@ def emit_decl_variant(decl):
                                              lvalues=lvalues)
         return constructor
 
+    def filter_decls(jit_decls, disable_autograd):
```

Comment:
I'm going to add another option to take a while list of ops to create custom mobile build so making this a separate function.